fast_pred/test/fastlightfm-test.cpp

```diff
@@ -22,12 +22,12 @@ TEST_CASE( "FastLightFM can warm-start predict (user 0)" ) {
     long top_k = 4;
     int number_of_users = 5;
```

Comment:
number_of_users is not really needed any more in this use case, is it?

Reply:
ah, sure,